std/math.d

```diff
@@ -328,6 +335,9 @@ template floatTraits(T)
             enum EXPPOS_SHORT = 0;
             enum SIGNPOS_BYTE = 0;
         }
+
+        import std.typecons : Tuple;
+        alias Integral = Tuple!(ulong, "significand", ushort);
```

Comment:
>I would be very surprised if `Tuple` doesn't pack data,

AFAICT, it doesn't. That's not a problem so far because the types are ordered appropriately, but might become one in the `BigEndian` case. The solution (if needed) is probably as simple as dropping `align(1):` in the `Tuple`'s body. This can be added as an option to existing `Tuple`, or as a small ad-hoc `private` type here; not sure which is better.

> >This is doable, but it is definitely more work than I'm ready to devote to supporting a format for which I have no simple and straightforward way of testing
>
>Yes, I know. I was merely questioning the `significand` accessor for the purpose of porting to `BigEndian` - this is something that I will get around to, but not today.

Come to think of it, that comment is a bit offensive. I'm sorry, didn't mean to.
Regarding `significand`, I only added it because there's a lot of code already in this module that manipulates the first `ulong` of `ieeeExtended` `real`s separately, so I thought that using a named accessor would look better and clearer than `integral[0]`. If that's causing so much trouble, I can drop it with no remorse.